src/setup/cortx_deploy

```diff
@@ -28,8 +28,9 @@ function show_usage {
     echo -e "                   DEBUG=1, INFO=2, WARNING=3, ERROR=4, FATAL=5"
     echo -e " CORTX-CONFIG-URL: URL of CORTX Configuration"
     echo -e "                   (Default is yaml:///etc/cortx/cluster.conf)"
-    echo -e " ENABLE MOCK     : Enable Mocking CORTX Deployment (Default: False)"
+    echo -e " ENABLE SKIP     : Skip check and forcefully execute config apply and bootstrap cmds."
```

Comment:
Skip Check is different from "force". We need "force". 

Skip means, dont check, skip it. 
Force means, check but ignore if check fails by issuing a warning.

Reply:
Done.